tidb-binlog/tidb-binlog-configuration-file.md

```diff
@@ -172,8 +172,12 @@ aliases: ['/docs-cn/dev/tidb-binlog/tidb-binlog-configuration-file/','/docs-cn/d
 
 ### initial-commit-ts
 
-* 指定从哪个 commit timestamp 之后开始同步。这个配置仅适用于初次开始同步的 Drainer 节点。如果下游已经有 checkpoint 存在，则会根据 checkpoint 里记录的时间进行同步。
-* 默认：`-1`。Drainer 会从 PD 得到一个最新的 timestamp 作为初始时间。
+* 指定从哪个 commit ts 之后开始同步。这个配置仅适用于初次开始同步的 Drainer 节点。如果下游已经有 checkpoint 存在，则会根据 checkpoint 里记录的时间进行同步。
```

Comment:
can we change `commit ts` to `[PD TSO](link xxx)`

Reply:
Actually, I didn't find the strict definition(link) of `TSO` in our document. 🤔️ we have lots of similar descriptions of this concept in many documents.

Reply:
```suggestion
* 指定从哪个事务提交时间点（事务的 commit ts） 之后开始同步。这个配置仅适用于初次开始同步的 Drainer 节点。如果下游已经有 checkpoint 存在，则会根据 checkpoint 里记录的时间进行同步。
```

Reply:
LOL